commands/diff/__tests__/__snapshots__/diff-command.test.js.snap

```diff
@@ -1,59 +1,52 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`DiffCommand passes diff exclude globs configured with --ignored-changes 1`] = `
-diff --git a/packages/package-1/package.json b/packages/package-1/package.json
-index SHA..SHA 100644
---- a/packages/package-1/package.json
-+++ b/packages/package-1/package.json
-@@ -1,5 +1,5 @@
- {
-   "name": "package-1",
--  "changed": 0,
-+  "changed": 1,
-   "version": "1.0.0"
- }
+"[0;31m------ [0m[0;1mpackages/package-1/package.json 100644[0m
```

Comment:
Your (global) git config seems to be enforcing colors no matter what? Should probably switch that to `auto` instead.